types/globaloffensive/index.d.ts

```diff
@@ -92,6 +92,10 @@ declare class GlobalOffensive extends EventEmitter {
         d?: string,
         callback?: (item: GlobalOffensive.ItemInfo) => void,
     ): void;
+    inspectItem(
```

Comment:
```suggestion
    /**
     * Sends the same request to the GC that the official client sends when you inspect an item.
     * If all parameters are correct and the GC is in a good mood, returns the item's data.
     * Using this for an item your account owns is useless as all the data is already available in `inventory`.
     *
     * The response will arrive in the callback and in the `inspectItemInfo` event.
     *
     * As of v2.1.0, the request will time out if no response is received in 10 seconds.
     * If this happens, `inspectItemTimedOut` will be emitted.
     *
     * @param owner The numeric SteamID or market listing ID of the owning Steam account or market listing,
     *              as a string; or an entire inspect link.
     * @param callback Called if all parameters are valid when Steam responds to us.
     *
     * @since v1.1
     */
    inspectItem(
```
Let's add the adjusted documentation for this overload as well :)